### ISSUE TITLE
Pyright fails to find `cookies` member of Django's `HttpResponse`

### ISSUE THREAD

**Issue by apollo13** [microsoft/pyright#5927]
**Describe the bug**
Pyright fails to discover members on `HttResponse` but only in strict mode, the following code works fine on mypy (no matter the mode).

**Code or Screenshots**
```python
from typing import reveal_type

from django.http import HttpResponse

response = HttpResponse()
reveal_type(response)
print(response.cookies["_up_method"].value)
```
`django` and `django-stubs` are installed. The relevant type in the stubs file is defined here: https://github.com/typeddjango/django-stubs/blob/87f48b3ae62cd983c72f2e9639e80a9a7c6e487f/django-stubs/http/response.pyi#L26 -- are the stubs somehow not used or am I doing something wrong?

The output:
```
/home/florian/sources/unpoly/python-unpoly/test.py
  /home/florian/sources/unpoly/python-unpoly/test.py:6:13 - information: Type of "response" is "HttpResponse"
  /home/florian/sources/unpoly/python-unpoly/test.py:7:7 - error: Type of "cookies" is unknown (reportUnknownMemberType)
  /home/florian/sources/unpoly/python-unpoly/test.py:7:7 - error: Type of "value" is unknown (reportUnknownMemberType)
  /home/florian/sources/unpoly/python-unpoly/test.py:7:7 - error: Argument type is unknown
    Argument corresponds to parameter "values" in function "print" (reportUnknownArgumentType)
3 errors, 0 warnings, 1 information 
```

**VS Code extension or command-line**
pyright command-line version 1.1.326

**Comment by erictraut** [comment]
Pyright isn't "failing to discover members" in this case. It's simply telling you that the evaluated type of the member is `Unknown`. In strict mode, the `reportUnknownMember` diagnostic check is enabled.

This appears to be a bug in the type stubs. The `response.pyi` type stub declares the type of `cookies` as `SimpleCookie`. This symbol comes from `cookie.pyi` and is defined as a variable `SimpleCookie: Any`. Variables are not allowed to appear within type annotations, so pyright treats it as an `Unknown` type. I suspect that the intent of the stub authors was to create a type alias (`SimpleCookie = Any`) or a class that derives from `Any` (`class SimpleCookie(Any): ...`).

**Comment by apollo13** [comment]
I see, sorry for the noise. Will see about getting this fixed upstream.
